app/assets/stylesheets/complaints.scss

```diff
@@ -0,0 +1,6 @@
+.summary-section {
+  border: 2px solid #dfe1e2;
+  border-radius: 5px;
+  padding: 0 1rem;
+  background-color: #fff;
```

Comment:
[suggestion] do we have a style guide to refer to for CSS?  I have historically alphabetized and tried to stick to the same units (px vs rem) with CSS where possible, but I am flexible if there are particular reasons you selected pixels and wanted background-color at the bottom, or if you all just don't think we ought to be concerned about it.

Reply:
There was no thought at all about order. I'm happy to alphabetize.

For units, I generally like to use `rem` for everything, but things like border and border-radius are such small values that `rem` gets a little silly to look correct. `0.15rem`

Reply:
Sounds like a good reason :)